wire-library/wire-runtime-swift/src/test/swift/AnyMessageTests.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Square Inc.
```

Comment:
```suggestion
 * Copyright 2022 Block Inc.
```

Reply:
Ah yea I saw a few variants of these around the codebase - I'll change to this one though